includes/defaults.inc.php

```diff
@@ -947,3 +947,6 @@
 $config['api']['cors']['maxage'] = '86400';
 $config['api']['cors']['allowmethods'] = array('POST', 'GET', 'PUT', 'DELETE', 'PATCH');
 $config['api']['cors']['allowheaders'] = array('Origin', 'X-Requested-With', 'Content-Type', 'Accept', 'X-Auth-Token');
+
+// The target maximum hostname length when applying the shorthost() function.
+$config['shorthost_target_length'] = 12;
```

Comment:
Please comment this out.

Did you check if setting it higher messes up any pages/graphs?

Reply:
I am running this in my environment with strings up to 25 chars long with no problem.
I will comment this out and leave a bit more description